packages/camera/camera_avfoundation/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.0
```

Comment:
This is a breaking version change in Dart's ecosyste, and would require manual adoption in the app-facing package. We should just use 0.9.17.

Reply:
Derp. Fixed!